### PR TITLE
Add accessibility examples

### PR DESCRIPTION
**Pull request by suminkimm**
## Summary
Added examples to RNTester for the AccessibilityInfo API for:
1. SetAccessibilityFocus()
2. isReduceMotionEnabled()
3. iOS Only
   a. isBoldTextEnabled()
   b. isGrayScaleEnabled()
   c. isInvertColorsEnabled()
   d. isReduceTransparencyEnabled()

## Changelog

[General] [Added] - Add new examples to the Accessibility API page

## Test Plan
iOS GIF

### CODE REVIEW

File: packages/rn-tester/js/components/ExamplePage.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ const styles = StyleSheet.create({
   examplesContainer: {
     width: ScreenWidth,
     flexGrow: 1,
+    flex:1
```
Comment (mkcode) [inline packages/rn-tester/js/components/ExamplePage.js:81]: Add trailing comma

File: packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js [hunk 2]
```diff
@@ -18,15 +18,18 @@ const {
   View,
   TouchableOpacity,
   TouchableWithoutFeedback,
+  findNodeHandle,
   Alert,
   StyleSheet,
+  Platform
```
Comment (mkcode) [inline packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js:24]:
```suggestion
  Platform,
```

File: packages/rn-tester/js/components/ExamplePage.js (hunk 1 above)
Comment (mkcode) [reply packages/rn-tester/js/components/ExamplePage.js:81]:
```suggestion
    flex:1,
```

**Comment by mkcode** [comment]
This looks good to me.

**Comment by suminkimm** [comment]
@rickhanlonii the only test failing is test_ios_unit_frameworks_hermes which is also failing on master!

**Pull request closed without merging**